### ISSUE TITLE
Activity cannot be started from background after beacon was detected

### ISSUE THREAD

**Issue by VictorCorchez** [AltBeacon/android-beacon-library-reference#65]
08-10 12:57:41.554 16404 16809 D MainActivity: Current region state is: INSIDE
08-10 12:57:41.554 16404 16809 D MainActivity: Got a didEnterRegion call
08-10 12:57:41.554 16404 16809 D MainActivity: Calling startActivity()
08-10 12:57:41.556  1289  3046 D OemSceneCallBlock: isCallBlockedWithUidIntent { act=android.intent.action.MAIN flg=0x18000000 cmp=<mainPackage>/<activityPackage>.MainActivity }, ResolveInfo{c0dfce1 <mainPackage>/<activityPackage>.MainActivity m=0x0}, false
08-10 12:57:41.557  1289  3046 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN flg=0x18000000 cmp=<mainPackage>/<activityPackage>.MainActivity} from uid 10756 pid 16404

This happens while testing on Android 10 and seems this could be the explanation:
https://developer.android.com/guide/components/activities/background-starts

**Comment by davidgyoung** [comment]
@VictorCorchez, yes, I believe your analysis of the problem is correct.  It is no longer possible for apps to launch activities from the background.

While the original version of this reference app did try to launch an activity from the background, I removed that and replaced it with a notification last year in the commit below.  Perhaps you are still using an old copy of the reference app?  If there is something posted today that does code to do this, please let us know.

https://github.com/AltBeacon/android-beacon-library-reference/commit/0183083517e915b345a2722dfae37267a9534fad

**Comment by VictorCorchez** [comment]
so is there is no other way of launching an activity from background without user interaction? I was thinking maybe this library could use the CompanionDevice approach.

**Comment by davidgyoung** [comment]
As of Android 10 I do not believe this is possible anymore per the link you provided in this issue.  I see nothing in [Companion Device Pairing](https://developer.android.com/guide/topics/connectivity/companion-device-pairing) that would help here

**Comment by VictorCorchez** [comment]
What about this: https://developer.android.com/reference/android/Manifest.permission#REQUEST_COMPANION_RUN_IN_BACKGROUND
?

**Comment by davidgyoung** [comment]
This repo has moved
